typings/xterm-player.d.ts

```diff
@@ -1,6 +1,52 @@
 /// <reference lib="dom"/>
 
+
 declare module 'xterm-player' {
+
+  import * as xterm from 'xterm'
+
+  /**
+   * An object containing theme settings of the player
+   *
+   * This is a subset of `xterm.ITerminalOptions`
+   */
+  export interface ITerminalOptions {
+    /**
+     * The font size used to render text
+     */
+    fontSize?: number
+
+    /**
+     * The font family used to render text
+     */
+    fontFamily?: string
+
+    /**
+     * The font weight used to render non-bold text
+     */
+    fontWeight?: xterm.FontWeight
+
+    /**
+     * The font weight used to render bold text
+     */
+    fontWeightBold?: xterm.FontWeight
+
+    /**
+     * The color theme of the terminal
+     */
+    theme?: xterm.ITheme
+  }
+
+  /**
+   * An object containing start up options for the player
+   */
+  export interface IPlayerOptions {
+    /**
+     * The options of the terminal
+     */
+    terminalOptions?: ITerminalOptions
+  }
```

Comment:
How about make it directly, instead of nested Object?
```typescript
export interface IPlayerOptions {
    /**
     * The font size used to render text
     */
    fontSize?: number

    /**
     * The font family used to render text
     */
    fontFamily?: string

    /**
     * The font weight used to render non-bold text
     */
    fontWeight?: xterm.FontWeight

    /**
     * The font weight used to render bold text
     */
    fontWeightBold?: xterm.FontWeight

    /**
     * The color theme of the terminal
     */
    theme?: xterm.ITheme
  }
```

Reply:
I make it nested because, a player contains a terminal, i.e. `player.option = terminal.option + some_other_options_about_player`, maybe you want to add some options that has nothing to do with terminal later.

Reply:
Ok, I will remove the nested structure.